docs/02_get_started/quickstart.md

```diff
@@ -1,15 +1,57 @@
 # Quickstart
 
-- [ ] How to setup a robotframework project with RobotCode
-  - [ ] Create Virtual Environment
-    - [ ] install robotframework
-    - [ ] install additional tools
-       - [ ] robotframework-robocop
-       - [ ] robotframework-tidy
-    - [ ] create your first suite/test
-    - [ ] run your first test
-
-    - [ ] configure your robotframework project
-      - [ ] default settings in robot.toml
-        - python-path = ["resources", "lib"]
-        - output-dir = "reports"
+Welcome to RobotCode! In this section we will get your Visual Studio Code set up and go through the basics: installing the extension, setting up your environment, and verifying everything is working smoothly.
+
+## Requirements
+Make sure you have **Python 3.8** or greater installed.
+
+## Installing RobotCode
+1. Open Visual Studio Code.
+2. Go to the **Extensions** tab (Shortcut key `CTRL + SHIFT + X`).
+3. Search for **RobotCode** and install the extension ![RobotCode Icon](../robotcode-logo.svg "RobotCode Icon"){.inline-icon} [RobotCode - Robot Framework Support](https://marketplace.visualstudio.com/items?itemName=d-biehl.robotcode "RobotCode Extension"). This will also install the **Python** and **Python Debugger** extensions.
+4. **(Optional)** Install the **Even Better TOML** extension for handling `.toml` files more effectively. We'll use a `robot.toml` file in the [Configuration](./configuration) section to set up our project settings.  
+
+After installation:
+   - Right-click on the RobotCode extension in the Extensions tab.
+   - Click **Add to Workspace Recommendations** to ensure that all team members working on this project are prompted to install the RobotCode extension, creating a consistent development environment for everyone.
+
+## Initialize an Empty Project
+
+1. **Create a `requirements.txt` file** in the root folder of your project and add the following dependencies:
+::: code-group
+```txt:line-numbers [requirements.txt]
+robotframework
+robotframework-tidy
+```
+:::
+
+2. **Set up your Python environment:**
+
+In this step we will let Visual Studio Code create a virtual environment using the selected Python version, install the dependencies listed in requirements.txt, and activate the virtual environment. This ensures that your project has all the necessary packages and an isolated environment for development.
+
+1. Click on **Create Environment** in Visual Studio Code. If this button isn't visible, you can alternatively open the Command Palette by pressing `CTRL + SHIFT + P`, then search for **Python: Create Environment**, and select it.
```

Comment:
Can you explain this a little bit more? Where is this `Create Environment` thing?

Reply:
After you create the requirements.txt file, Visual Studio Code might pop up a button called "Create Environment" at the bottom right corner of the IDE. You can see an example of this button at the 10:26 mark in the Robotcode demonstration video: https://www.youtube.com/live/7Uad_250YuI?si=2tCLFHEyM2s5CU-X&t=626